AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -98,12 +105,41 @@ - (instancetype)init
   return [self init];
 }
 
+#pragma mark - 
+
+- (void)cacheSublayouts
```

Comment:
Since this functionality is only used in one place, and it would be unusual / maybe dangerous to call more than once — what do you think about putting these lines directly in the initializer method that uses them?

Otherwise, let's make it a property -- and in setCacheSublayouts: we can either set up the array or destroy it.